flask_security/cache.py

```diff
@@ -0,0 +1,38 @@
+from hashlib import md5
+
+from flask import current_app
+from cachetools import TTLCache
+
+
+class VerifyHashCache:
```

Comment:
I think to be 2.7 compatible this will need to be: VerifyHashCache(object):

Reply:
this is not an issue for me. 

![image](https://user-images.githubusercontent.com/5399780/57175167-0c53d780-6e49-11e9-849f-8e2da89111d7.png)

Reply:
The drawbacks of not doing so is that some class related feature are not available with a class declared this way and some of these points are described here https://stackoverflow.com/questions/4015417/python-class-inherits-object

The cache object I made does not uses any of these feature, so this is not required, but I guess for the sake of python the  inheritance should be written.

On the other hand 2020 is comming very fast :)